src/flamegraph/mod.rs

```diff
@@ -102,6 +102,9 @@ pub struct Options<'a> {
     /// If `None`, the background color will be selected based on the value of `colors`.
     pub bgcolors: Option<color::BackgroundColor>,
 
+    /// The color of UI text such as the search and reset view button. Defaults to black
+    pub uicolor: color::Color,
```

Comment:
Given what it does, should this be called `textcolor` instead?

Reply:
The answer is similar to the above. This doesn't change all the text, only the free-standing text outside the actual flame graph